src/main/java/io/github/dsheirer/gui/playlist/streaming/BroadcastifyCallEditor.java

```diff
@@ -177,6 +188,22 @@ protected GridPane getEditorPane()
             GridPane.setConstraints(getMaxAgeTextField(), 1, row);
             mEditorPane.getChildren().add(getMaxAgeTextField());
 
+            Label testEnabledLabel = new Label("Scheduled Connection Test");
```

Comment:
Doug, could you rename the labels for these two additions to something that would be more intuitive to a general user, so we can avoid questions like 'what does this do'?

Since the feature seems more like a 'keep-alive' than a test, maybe something like:

'Send Periodic Keep-Alive' and 'Ping Interval'

Reply:
Labels and logging updated to use new terms.